runbotics/runbotics-desktop/src/core/bpm/runtime/runtime.service.ts

```diff
@@ -685,4 +692,23 @@ export class RuntimeService implements OnApplicationBootstrap, OnModuleDestroy {
         content.isSequenceFlow &&
         content.sourceId.includes('Gateway_')
     );
+
+    private determineCredentialsForAction(
+        credentialId: string | undefined,
```

Comment:
or null. And there should be type CredentialId used. define it if it's not there yet